ui/components/GithubDeviceAuthModal.tsx

```diff
@@ -117,7 +117,7 @@ function GithubDeviceAuthModal({
 
       <Flex wide center>
         {codeLoading || !codeRes ? (
-          <CircularProgress />
+          <CircularProgress size={143} />
```

Comment:
Sort of a random value here. Can we pick some sizes that will keep things consistent? Small, Medium, Large?

Reply:
It's the value only because that's the exact size of the text that replaces it - I could wrap both in a div with a set size which would definitely be better